tools/gitleaks/config.toml

```
@@ -49,7 +49,7 @@
 	[rules.allowlist]
 		regexes = [
 			'''\"image/png\": \"[0-9a-zA-Z\/+]{1,}(=){0,2}\\n\"''',
-			'''sonarsource/sonarcloud-github-action@'''
+			'''uses: [a-zA-Z0-9_\/-]*@\b[0-9a-f]{40}\b'''
```

I don’t think the first character class should include backslashes, and the first `\b` isn’t necessary (the boundary is already fully defined):

```suggestion
			'''uses: [a-zA-Z0-9_/-]*@[0-9a-f]{40}\b'''
```

---

I was thinking about escaping the slash, not matching backslashes, but I see now that I set my regex tester go Go it doesn't seem to be necessary. Not needing the first `\b` makes sense, thanks. Changed both in the latest.

---

Right, I wondered whether that was the intent — `/` isn’t a special character in regular expressions, and in any case, most special characters don’t need to be escaped in character classes (which is a trick often used to make escapes easier to ready; for example `[.]` instead of `\\.` in scenarios where double escapes are required).